src/Raven.Server/Documents/Indexes/Persistence/Corax/CoraxQueryEvaluator.cs

```diff
@@ -0,0 +1,1277 @@
+﻿using System;
```

Comment:
This is leftover. Currently, we're using `CoraxQueryBuilder`.

Reply:
@redknightlois can you please delete that file?